pkg/policy/policies.go

```diff
@@ -66,8 +66,9 @@ func (ps *Policies) PIDFilterableInUserSpace() bool {
 	return ps.pidFilterableInUserSpace
 }
 
-// ContainerFilterEnabled returns a bitmask of policies that have at least one
-// container filter type enabled
+// ContainerFilterEnabled returns a bitmask of policies that have at least one container filter type
+// enabled. TODO: make sure the stores are also atomic (an atomic load is only protecting the read
+// from context switches, not from CPU cache coherency issues).
```

Comment:
Currently there's no issue since the single store is at first Policies.Compute(): https://github.com/aquasecurity/tracee/blob/5d28f5a42811ed333db240289992148fd47c68f2/pkg/policy/policies.go#L77-L82

But the comment warning about this is very welcome.

Reply:
Soon there will be with dynamic changes, thus the comment.